README.md

```diff
@@ -0,0 +1,26 @@
+# Lissi Connect through NodeJS
+
+This express application presents a working example of a Single Sign-On method through lissi-Connect using `passportJS` and `openid-client` strategy.
```

Comment:
You have at least four different ways for writing Lissi Connect:
- LissiConnect
- lissi-Connect
- Lissi Connect
- `Lissi Connect`

Please commit to one. I am suggesting either `lissi-connect` or Lissi Connect.